docs/tips/multitarget.md

```diff
@@ -0,0 +1,44 @@
+# Multi-target build
```

Comment:
Please copy the file to i18n folder too. https://github.com/tensorchord/envd-docs/tree/main/i18n/zh/docusaurus-plugin-content-docs/current

Reply:
done